public/app/core/components/AppChrome/TopBar/InviteUserButton.tsx

```diff
@@ -9,9 +9,10 @@ import { AccessControlAction } from 'app/types';
 import { NavToolbarSeparator } from '../NavToolbar/NavToolbarSeparator';
 
 export function InviteUserButton() {
+  config.externalUserMngLinkUrl = '(true)';
```

Comment:
don't have context - what does this do? 🤔

Reply:
@ashharrison90 sorry, this was just to test the invite button temporarily, will revert